packages/client/src/InfluxDBClient.ts

```diff
@@ -0,0 +1,134 @@
+import WriteApi from './WriteApi'
+import {Transport} from './transport'
+import WriteApiImpl from './impl/WriteApiImpl'
+// replaced by ./impl/browser/FetchTransport in browser builds
+import TransportImpl from './impl/node/NodeHttpTransport'
+import {ClientOptions, QueryType, WriteOptions} from './options'
+import {IllegalArgumentError} from './errors'
+import {NotArrayLike, NotPointRecord, Point, PointRecord} from './Point'
+import {convertTime} from './util/time'
+import {isArrayLike, isDefined} from './util/common'
+import QueryApi from './QueryApi'
+import QueryApiImpl from './impl/QueryApiImpl'
+
+export default class InfluxDBClient {
+  private readonly _options: ClientOptions
+  private readonly _writeApi: WriteApi
+  private readonly _queryApi: QueryApi
+  readonly transport: Transport
+
+  /**
+   * Creates influxdb client options from an options object or url.
+   * @param options - client options
+   */
+  constructor(options: ClientOptions | string) {
```

Comment:
Please remove the support for `string`. It will be more consistent across v3 clients.